backend/lib/services/infrastructureSecrets.js

```diff
@@ -168,27 +168,13 @@ async function getInfrastructureSecrets ({ secretBindings, cloudProfileList, sec
     .value()
 }
 
-async function getCloudProviderKind (cloudProfileName) {
-  const cloudProfile = await cloudprofiles.read({ name: cloudProfileName })
-  const cloudProviderKind = _.get(cloudProfile, 'metadata.cloudProviderKind')
-  return cloudProviderKind
+function getCloudProviderKind (name) {
+  const cloudProfile = cloudprofiles.read({ name })
+  return _.get(cloudProfile, 'metadata.cloudProviderKind')
 }
 
-async function getProjectByNamespaceIgnoreNotFound (client, namespace) {
-  let object
-  try {
-    object = await client.getProjectByNamespace(namespace)
-  } catch (err) {
-    if (!isHttpError(err, 404)) {
-      throw err
-    }
-  }
-  return object
-}
-
-async function getProjectMap (client, namespaces) {
-  const projectPromises = _.map(namespaces, namespace => getProjectByNamespaceIgnoreNotFound(client, namespace))
-  const projects = await Promise.all(projectPromises)
+function getProjectMap (client, namespaces) {
```

Comment:
update the comment in `list` function that calls `getProjectMap`

Reply:
✔️